### PR TITLE
add avivator container

### PR DESCRIPTION
**Pull request by sunyi000**
1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [x] LABEL about.documentation   
  - [x] LABEL about.license_file

### CODE REVIEW

**Comment by sunyi000** [comment]
@osallou could you please review it? Thank you very much

File: avivator/0.12.3/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:20.04 as builder
+
+LABEL base_image="ubuntu:20.04, nginx:latest"
+LABEL version="1"
+LABEL software="Viv/avivator"
```
Comment (osallou) [inline avivator/0.12.3/Dockerfile]: *software* mus tbe same as directory name ie avivator

**Review by osallou — CHANGES_REQUESTED** [review]

**Comment by osallou** [comment]
Hi,
labels must be oneline, else interpretor unfortunatly fail...

should be:

    LABEL base_image="ubuntu:20.04, nginx:latest" \
     version="1" \
     software="avivator" \
     software.version="0.12.3" \
     about.summary="Avivator is a lightweight batteries-included WebGL viewer for remote imaging data, built around Viv. " \
     about.home="https://github.com/hms-dbmi/viv/tree/master/avivator" \
     about.license="MIT License" \
     about.license_file="https://raw.githubusercontent.com/hms-dbmi/viv/master/LICENSE" \
     about.documentation="http://viv.gehlenborglab.org/" \
     maintainer="sunyi000@gmail.com"

**Comment by sunyi000** [comment]
> Hi, labels must be oneline, else interpretor unfortunatly fail...
> 
> should be:
> 
> ```
> LABEL base_image="ubuntu:20.04, nginx:latest" \
>  version="1" \
>  software="avivator" \
>  software.version="0.12.3" \
>  about.summary="Avivator is a lightweight batteries-included WebGL viewer for remote imaging data, built around Viv. " \
>  about.home="https://github.com/hms-dbmi/viv/tree/master/avivator" \
>  about.license="MIT License" \
>  about.license_file="https://raw.githubusercontent.com/hms-dbmi/viv/master/LICENSE" \
>  about.documentation="http://viv.gehlenborglab.org/" \
>  maintainer="sunyi000@gmail.com"
> ```

thanks for reviewing it. I just made the changes, but it seems still looking for 'software' key in labels. Did I still miss anything?
thank you

**Comment by osallou** [comment]
arr, multi-stage build

LABEL should be defined in "final" image

     FROM nginx:latest
    
     LABEL base_image="ubun....

**Comment by sunyi000** [comment]
thank you very much. I just fixed the license text. 
I ignored the comment about extra.identifier.biotools as it's not defined in bio.tools

**Comment by osallou** [comment]
for bio.tools it is not mandatory, only a warning/check

**Comment by osallou** [comment]
and thanks for contribution

**Pull request merged**